src/Microsoft.Performance.SDK/Processing/ColumnBuilding/ColumnVariantIdentifier.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Performance.SDK.Processing.ColumnBuilding;
+
+/// <summary>
+///     Represents a identifier for a column variant.
+/// </summary>
+/// <param name="Guid">
+///     The unique identifier for the column variant.
```

Comment:
>The unique identifier for the column variant. [](http://example.com/codeflow?start=8&length=45)

Does this only need to be unique within a column - meaning it can be used in other columns without an issue?

Reply:
I think by definition these identifiers need to be **globally** unique.

Reply:
hmm, what do we do for scenarios where a column projection is reused across multiple tables?

Reply:
The projection itself can be re-used, but if the same projection is being used for multiple columns' variants they need different GUIDs.

Reply:
Hmm, thinking about that scenario more, I can see a benefit of letting different columns have the same GUIDs for similar variants (e.g. the "project as datetime" variant).

I think it's OK to duplicate those GUIDs. We should only really be searching for a guid per-column anyway. I'll update the docs.